gridkit/hex_grid.py

```diff
@@ -522,6 +522,55 @@ def is_aligned_with(self, other):
         )
         return aligned, reason
 
+    def to_crs(self, crs, resample_method="nearest"):
+        """Transforms the
+
+        The ``crs`` attribute on the current GeometryArray must
+        be set.  Either ``crs`` or ``epsg`` may be specified for output.
+        This method will transform all points in all objects.  It has no notion
+        or projecting entire geometries.  All segments joining points are
+        assumed to be lines in the current projection, not geodesics.  Objects
+        crossing the dateline (or other projection boundary) will have
+        undesirable behavior.
+
+        Parameters
+        ----------
+        crs :class:`pyproj.CRS`
+            The value can be anything accepted
+            by :meth:`pyproj.CRS.from_user_input() <pyproj.crs.CRS.from_user_input>`,
+            such as an epsg integer (eg 4326), an authority string (eg "EPSG:4326") or a WKT string.
+
+        Returns
+        -------
+
```

Comment:
Add description on why resample_method is not used
Add return type
Add see also